frontend/src/components/ExpensePresentation.js

```diff
@@ -1,28 +1,19 @@
 import React from "react";
-import { Table } from "react-bootstrap";
+import "../styles/Expenses.css";
 
 class ExpensePresentation extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      fixed: 2000,
-      irregular: 800,
+      expenses: 2800,
     };
   }
   render() {
     return (
-      <Table>
-        <tbody>
-          <tr>
-            <td>Total Fixed Expense</td>
-            <td>{this.state.fixed} €</td>
-          </tr>
-          <tr>
-            <td>Total Irregular Expense</td>
-            <td>{this.state.irregular} €</td>
-          </tr>
-        </tbody>
-      </Table>
+      <div className="expPres">
+        <div>Total Expense</div>
+        <div>{this.state.expenses}</div>
```

Comment:
The "€" sign is missing from this row.

Reply:
Did put the "€" sign

Reply:
Let's not add it in the state expenses property, we need to use that as a number. It should go to the html structure, like so:
\<div\>{this.state.expenses} €\</div\>

Reply:
Resolved in 98ae7b35ef0c274798064fc2b47af099f5e6cfa2